Packs/Intezer/Integrations/IntezerV2/IntezerV2.py

```diff
@@ -1,22 +1,34 @@
+from http import HTTPStatus
+from typing import Callable
+from typing import Dict
+from typing import List
+from typing import Union
+
 import demistomock as demisto
+import requests
 from CommonServerPython import *
 from CommonServerUserPython import *
-
-''' IMPORTS '''
-
-import requests
-
+from intezer_sdk import consts
+from intezer_sdk.analysis import Analysis
+from intezer_sdk.analysis import get_analysis_by_id
+from intezer_sdk.analysis import get_latest_analysis
+from intezer_sdk.api import IntezerApi
+from intezer_sdk.errors import AnalysisIsAlreadyRunning
+from intezer_sdk.errors import AnalysisIsStillRunning
+from intezer_sdk.errors import FamilyNotFoundError
+from intezer_sdk.errors import HashDoesNotExistError
+from intezer_sdk.errors import IntezerError
+from intezer_sdk.family import Family
+from intezer_sdk.sub_analysis import SubAnalysis
+from requests import HTTPError
+
+''' CONSTS '''
 # Disable insecure warnings
 requests.packages.urllib3.disable_warnings()
 
-API_KEY = demisto.getParam('APIKey')
-SERVER_URL = 'https://analyze.intezer.com/api'
-API_VERSION = '/v2-0'
-BASE_URL = SERVER_URL + API_VERSION
 IS_AVAILABLE_URL = 'is-available'
 ERROR_PREFIX = 'Error from Intezer:'
 ACCEPTABLE_HTTP_CODES = {200, 201, 202}
-USE_SSL = not demisto.params().get('insecure', False)
```

Comment:
Please handle the ssl param, and also please if you can do it in the main function.